tests/emukit/core/test_stopping_conditions.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+import mock
+
+from emukit.core.loop import (ConvergenceStoppingCondition,
+                              FixedIterationsStoppingCondition,
+                              LoopState,
+                              StoppingCondition)
+
+
+class DummyStoppingCondition(StoppingCondition):
+    def should_stop(self, loop_state: LoopState) -> bool:
+        pass
+
+
+def test_fixed_iteration_stopping_condition():
+    stopping_condition = FixedIterationsStoppingCondition(5)
```

Comment:
might be worth making a variable here for the number 5, e.g. iter_stop = 5, and then test if it doesn't stop at iter_stop-1, but stops at iter_stop. In the current test it is not clear if it stops too early by accident at not-stopping is only tested for 0<<stop_iter

Reply:
good point about not testing anything in between 0 and 5!